content/pages/en/media.mdx

```diff
@@ -1,3 +1,31 @@
+---
+title: "Media"
+hero.title: "Media"
+---
+
+## Copywriting
+
+### Common to all languages
+
+- The name of the product is “Open Terms Archive”.
+- Open Terms Archive is a brand name, not a common noun. It does not have a determiner.
+- It is singular. It is not “Open Terms Archives”.
+- It has spaces. It is not “OpenTermsArchive”.
+- As often as possible, non-breakable spaces should be used to separate the three words, in order to avoid line breaks in the middle of the name.
+- While the “OTA” abbreviation can be used in internal documents and discussions, it is not used publicly (at least not by us).
+
+### Exemples
+
+```diff
+- OTA enables activists to identify misleading terms.
+- The Open Terms Archive enables activists to identify misleading terms.
+- OpenTermsArchive enables activists to identify misleading terms.
++ Open Terms Archive enables activists to identify misleading terms.
+```
```

Comment:
> Is this rendered properly in the UI? 🙂 Providing a screenshot would be really helpful for the review process.

Reply:
![image](https://user-images.githubusercontent.com/364319/204253945-eb9dab95-82cd-46df-af2b-aad93c097a02.png)

Reply:
Thanks! With this rendering, I believe it is at best confusing, at worst misleading for readers. Very few people would realise that this `+` / `-` prefix is a diff and means that the first three examples should NOT be used and the last one should. We need to change this into words, with something like “do not do this”, “do this instead”.

Reply:
Also, wouldn't it be more understandable if the gray background included the copywriting rules? 🙂

Reply:
> Thanks! With this rendering, I believe it is at best confusing, at worst misleading for readers. Very few people would realise that this + / - prefix is a diff and means that the first three examples should NOT be used and the last one should. We need to change this into words, with something like “do not do this”, “do this instead”.

Agree with you, I made the following improvements for the readibility of the copywriting rules exemples:

<img width="768" alt="image" src="https://user-images.githubusercontent.com/364319/204836356-f6189182-8c1a-4e0e-83d4-04030cd7797e.png">

> Also, wouldn't it be more understandable if the gray background included the copywriting rules? 🙂

I also think we should make the different sections clearer but I can't do it simply and quickly because the copywriting rules and the press reviews are part of the same Markdown file.

Reply:
😍